libs/vertexai/langchain_google_vertexai/embeddings.py

```diff
@@ -109,6 +109,9 @@ def __init__(
         self.instance[
             "embeddings_task_type_supported"
         ] = not self.client._endpoint_name.endswith("/textembedding-gecko@001")
+        self.instance["dimensionality_supported"] = self.client._endpoint_name.endswith(
```

Comment:
could you move this logic to _utils, please, so that we have all the logic determening model's behaviour at least at one place (and we should introduct model family or smth like this).

Reply:
I've added ModelFamily notion, let's use it, please.